sphinx/themes/basic/search.html

```diff
@@ -15,7 +15,7 @@
     <script src="{{ pathto('_static/language_data.js', 1) }}"></script>
 {%- endblock %}
 {% block extrahead %}
-    <script src="{{ pathto('searchindex.js', 1) }}" defer></script>
+    <script src="{{ pathto('searchindex.js', 1) }}" defer="defer"></script>
```

Comment:
For the future: this is done so that it also validates against XML (and we can parse the content using `xml` native library instead of `lxml`).

Also for the future: this is mentioned by the HTML 4 specifications (https://www.w3.org/TR/1998/REC-html40-19980424/intro/sgmltut.html#h-3.3.4.2) which remain true for HTML 5.

Reply:
do you want this mentioned anywhere in the actual code? or its ok just to have it mentioned in this PR for prosperity

Reply:
Mmh.. no it's fine if you mention it in the commit message.